Neos.Fusion/Resources/Private/Fusion/Root.fusion

```diff
@@ -145,7 +143,7 @@ prototype(Neos.Fusion:UriBuilder) {
 #   class = 'some-class'
 # }
 #
-prototype(Neos.Fusion:Augmenter) < prototype(Neos.Fusion:Component) {
```

Comment:
Huh, this is unrelated, strictly speaking. Also, I (personally) dislike prototypes that "have no type", I always wonder what they actually are…

Why was this change done?

Reply:
youre right - this change was just because i saw this laying around - will revert ...

 it is just totally useless to have inheritance + an own `@class`
the `@class` will overrule the inheritance and we dont need inheritance in this case as `Neos.Fusion:Augmenter` has nothing to do with Fusion:Component

Reply:
`prototype(Neos.Fusion:Component)` would also "have no type" ^^

Reply:
Oh, yes. I should have ntoiced the `@class`… fine with me then.